_includes/hub_researcher_tags_and_cards.html

```diff
@@ -1,7 +1,27 @@
 <nav class="navbar navbar-expand-lg navbar-light research-menu">
   {% include hub_model_tags.html category="researchers" %}
+  <div class="hub-select-container col-3">
+
+    {% if page.compact == true && category="researchers"%}
```

Comment:
I don't think you need `&& category="researchers"` here. Since `hub_researcher_tags_and_cards` is only being called from the research index and compact research index pages, we know the category is researchers.